allennlp/training/metrics/boolean_accuracy.py

```diff
@@ -34,7 +34,6 @@ def __call__(self,
         mask: ``torch.Tensor``, optional (default = None).
             A tensor of the same shape as ``predictions``.
         """
-        # Get the data from the Variables.
         predictions, gold_labels, mask = self.unwrap_to_tensors(predictions, gold_labels, mask)
```

Comment:
This method feels like it should be renamed to something like `detach_and_move_to_cpu` (or something similar but more compactly named...), and maybe just moved to `nn.util`.